cli/command/secret/create_test.go

```diff
@@ -0,0 +1,126 @@
+package secret
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/cli/internal/test"
+	"github.com/docker/docker/pkg/testutil/assert"
+	"github.com/docker/docker/pkg/testutil/golden"
+	"github.com/pkg/errors"
+)
+
+const secretDataFile = "secret-create-with-name.golden"
+
+func TestSecretCreateErrors(t *testing.T) {
+	testCases := []struct {
+		args             []string
+		secretCreateFunc func(swarm.SecretSpec) (types.SecretCreateResponse, error)
+		expectedError    string
+	}{
+		{
+			args:          []string{"too_few"},
+			expectedError: "\"create\" requires exactly 2 argument(s)",
```

Comment:
Thank you for the review. The PR is now updated (removed references to the command name from the tests expected errors).